src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -63,7 +67,8 @@ std::string Notification::toJson()
   jh.addString("callback", this->callback);
   if (!this->throttling.isEmpty())
   {
-    jh.addRaw("throttling", this->throttling.render(JSON,"",false));
+    //jh.addRaw("throttling", this->throttling.render(JSON,"",false));
```

Comment:
I needed to change this due to the current code was producing a wrong JSON token:
```
"throttling":"throttling" : "FIXME"
```

Maybe it is a way of using the Throttling render() that fix it, without needing to composed using the throttling.get().